cypress/support/e2e.ts

```diff
@@ -30,3 +30,7 @@ declare global {
     }
   }
 }
+
+Cypress.on("uncaught:exception", (err, _runnable) => {
+  return false;
+});
```

Comment:
What is this needed for?

Reply:
When navigating to the MAAS docs, there's an error that says `canonicalGlobalNav is not defined`, along with a few other scripts which are not instrumental to our test. Once this exception is thrown, the test fails automatically. This prevents the test from failing because of the absence of those scripts